lms/djangoapps/discussion/django_comment_client/utils.py

```diff
@@ -1076,3 +1079,20 @@ def is_content_authored_by(content, user):
         return int(content.get('user_id')) == user.id
     except (ValueError, TypeError):
         return False
+
+
+def sanitize_body(body):
+    """
+    Return a sanitized version of the body with dangerous Markdown removed.
+
+    This is possibly overly broad, and might tamper with legitimate posts that
+    contain this code in fenced code blocks. As far as we can tell, this is an
```

Comment:
Nit/Question: Hmm - good point. Do you think we should be URL-encoding/escaping these links instead? In other words, still displaying them but not as clickable links?

Reply:
I feel like even that amount of munging could break something in a fenced block, unless the parsing gets a lot smarter. So I'm hoping that this is just rare enough where it's not going to come up in real life. :-/